sdk/rust/crates/dagger-codegen/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![deny(warnings)]
+//#![deny(warnings)]
```

Comment:
:eyes: is this intentional?

Reply:
Woops nope. I've got another pr with some changes to this and I didn't manage to get that in. I'll get it fixed =D